Update tube_tipper.scad

sebkopf opened the pull request:
@jen-reeve 
easiest to trouble shoot in a pull request since additional commits you make to your branch will just automatically become part of this pull request

i'll leave in line comments below

Commit:
Update tube_tipper.scad

sebkopf commented on tube_tipper.scad:
```diff
@@ -57,9 +57,9 @@ module tube_tipper(number_tubes = 5) {
+            0.25 * tube_spacing])
           rotate([0, 71, 0])
-          cylinder(d = 0.5 * tube_diameter, h = 2 * wall_thickness);
+          cylinder(d = 0.5 * (tube_diameter + tube_z_stretch), h = 2 * wall_thickness);
```
I don't think this works, it will not introduce the right kind of stretch if you add it to the diameter. Better to add a `resize([0, 0, tube_diameter + tube_z_stretch])` above the rotate instead.

jen-reeve replied:
Made this change, in the next commit

sebkopf commented on tube_tipper.scad:
```diff
@@ -22,11 +22,11 @@ module tube_tipper(number_tubes = 5) {
   tube_diameter = 18.2; // diameter of the tubes
   tube_z_stretch = 1; // how much to stretch in vertical direction to account for printing collapse
```
looks to me like the tube z stretch works just fine. Easiest way to see if something works in the rendering is to exaggerate it greatly. Try `tube_z_stretch = 1;` and see what happens

jen-reeve replied:
Yup, exaggeration does make it seem like this is working, I'll kick it up a touch since it seems to not be on strong enough.

sebkopf commented on tube_tipper.scad:
```diff
@@ -40,7 +40,7 @@ module tube_tipper(number_tubes = 5) {
           translate([-wall_thickness,
             (y - (number_tubes+1)/2) * (tube_diameter + tube_spacing),
-            -0.5 * (tube_spacing + tube_diameter) + (0.5 + x) * (tube_diameter + tube_z_stretch + tube_spacing)])
+            -0.5 * (tube_spacing + 0.75 * tube_diameter) + (0.5 + x) * (tube_diameter + tube_z_stretch + tube_spacing)])
```
it seems this translate does not yet move the bottom row down more, wasn't this already part of the last printing run? 
sidenote: I think for the 20degree angle to keep working you should only move the bottom holes lower, the top ones must stay where they are, right?

jen-reeve replied:
Moving the bottom row down was part of the last test, I think I need to move it a little back up so that the bottom of the hole is sitting right at the base.

On the sidenote: yes, that is how angles work, fixing now.

Commit:
continued tube tipper work

Fixed several of the comments from the pull request (mostly z stretch related). Still need to fix the height of the top row.

Commit:
fixing tube tipper top row height

I think I got the top row back into the right spot, will do a small test with the 5 tube
